content/en/docs/contribute/participate/roles-and-responsibilities.md

```diff
@@ -74,8 +74,8 @@ After submitting at least 5 substantial pull requests and meeting the other
    [SIG Docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs).
 
    {{< note >}}
-   Don't send a direct email or Slack direct message to an individual
-   SIG Docs member. You must request sponsorship before submitting your application.
+   Don't send a direct email or Slack direct message to individual
+   SIG Docs members randomly. You must request sponsorship before submitting your application.
```

Comment:
nit: I think the original text is clear. What would you like to change?

Reply:
The original text is "don't send any DM". I think it should be change to "don't spam others".

The motivate of changes is stated in PR description, not sure how to make the new text clear too.

Reply:
I think the note about `Don't send a direct email to individual SIG Docs members` should remain.
Is it okay to send a single Slack message to another contributor?  Probably, especially after you have reached out on the main DOCS channels. Repeatedly sending requests directly to another contributor is not recommended (or acceptable).

Reply:
Hum... Okay, let me revert this part.